src/mkCommand.nix

```diff
@@ -1,5 +1,13 @@
 {nixpkgs}: let
-  writeShellScript = system: nixpkgs.legacyPackages.${system}.writeShellScript;
-  mkCommand = system: args: args // {command = writeShellScript system "${args.name}" args.command;};
+  mkCommand = system: args: let
+    inherit (nixpkgs.legacyPackages.${system}) pkgs;
+  in
+    args
+    // {
+      command = (pkgs.writeShellScript "${args.name}" args.command).overrideAttrs (_:
```

Comment:
https://github.com/nlewo/nix2container/pull/65 was merged :smile:

Reply:
Nice I updated the input and remove this logic